en/controllers/request-response.rst

```diff
@@ -1067,6 +1067,26 @@ criteria are met:
     is very application specific. We recommend you build your own ``CORSMiddleware``
     if you need one and adjust the response object as desired.
 
+Running logic after the Response has been sent
+----------------------------------------------
+
+In fastcgi based environments you can listen to the ``Server.terminate`` event
+to run logic **after** the response has been sent to the client. Make sure your
+application's **webroot/index.php** contains the following::
```

Comment:
I couldn't write an exact replacement, but think we need to spell out that the changes must be added around the existing emit() call.

New applications created from the app template will include this.